addon/normalizer.js

```diff
@@ -38,39 +38,48 @@ export default class Normalizer {
     const type = this.primaryModelClass.modelName;
     const isCollection = false;
 
-    this._extractSingle(resourcePayload, type, isCollection);
+    this._extractSingle(resourcePayload, type, type, isCollection);
```

Comment:
`type, type` is very confusing. Looks like a type. This would be a lot more understandable if you made a `key` variable.

```
const key = type;
this._extractSingle(resourcePayload, key, type, isCollection);
```

But I do not think the `key` is needed, especially since it is always the same as the `type`. It gets passed into the Resource constructor, but `Resource` does not appear to internally use the `key` either.